src/components/List.js

```diff
@@ -157,17 +157,17 @@ const List = ({ token }) => {
                 <i class="fas fa-cart-plus"></i>
                 Add Item
               </a>
-              <label htmlFor="search-bar">Filter Items</label>
-              <br />
-              <input
-                type="text"
-                name="search-bar"
-                id="search-bar"
-                placeholder="Start typing here..."
-                value={searchInput}
-                onChange={handleSearchChange}
-              />
-              <input type="reset" onClick={handleClear} />
+              <Box display="flex" alignItems="center">
```

Comment:
Nice use of the Material-UI components! 👍 
Now that the input is taller, it really calls attention to the fact that the reset button is much shorter than the text input.
Have you thought about swapping out the reset button for a Material-UI Button component, too?

Reply:
@chrisdopuch You read out minds!